docs/guide/migrating-to-version-9.md

```diff
@@ -139,6 +139,23 @@ Kongponents styles are no longer designed to be utilized standalone, separately
 
 ### KDateTimePicker
 
+#### Props
+
+* `minuteIncrement` Deprecated in favor of the `rules` prop. [Refer to v-calendar documentation](https://vcalendar.io/datepicker/time-rules.html)
```

Comment:
question: should we deprecate, or just remove now? 

If we deprecate, we can't remove it until the _next_ major release. 

If you're going to manually pull in this PR in `khcp-ui` and `konnect-ui-apps` as needed (and ensure usage is updated in Mesh if it exists, and `konnect-portal`, and Kong Manager `kong-admin`) then we can just remove it now.

You can reach out to @kleinfreund, @sumimakito, and @andrewwylde to check for usage in other repos.

Reply:
konnect-ui-apps PR: https://github.com/Kong/konnect-ui-apps/pull/1869
konnect-portal PR: https://github.com/Kong/konnect-portal/pull/336

Reply:
@adamdehaven regarding `minuteIncrement` - we only had one instance of `:minute-increment="5"` in the Service Hub TableFilter, and that was not working anyway because v-calendar dropped support for it. 

This PR actually fixes the 5 minute increment, by internally setting it to the values we want:
https://github.com/Kong/kongponents/pull/1881/files#diff-dca1c68377d6ca31ada7b247da920e36d405a688da991bef15d03a2059c00d05R304

Reply:
Hi @mihai-peteu - It seems that Kong Manager does not use the `<KDateTimePicker />` after searching throughout the codebase, so it should have no impact on Kong Manager.